packages/mui-material/src/ScopedCssBaseline/ScopedCssBaseline.js

```diff
@@ -21,8 +21,8 @@ const ScopedCssBaselineRoot = styled('div', {
   name: 'MuiScopedCssBaseline',
   slot: 'Root',
   overridesResolver: (props, styles) => styles.root,
-})(({ theme }) => ({
-  ...html,
+})(({ theme, enableColorScheme }) => ({
+  ...html(theme, enableColorScheme),
```

Comment:
I do not found how to pass `enableColorScheme` to this component, and at the same time avoid the following warning
> Warning: React does not recognize the `enableColorScheme` prop on a DOM

Reply:
```diff --git a/packages/mui-material/src/ScopedCssBaseline/ScopedCssBaseline.js b/packages/mui-material/src/ScopedCssBaseline/ScopedCssBaseline.js
index f67c9d661d..c833821012 100644
--- a/packages/mui-material/src/ScopedCssBaseline/ScopedCssBaseline.js
+++ b/packages/mui-material/src/ScopedCssBaseline/ScopedCssBaseline.js
@@ -34,7 +34,7 @@ const ScopedCssBaselineRoot = styled('div', {

 const ScopedCssBaseline = React.forwardRef(function ScopedCssBaseline(inProps, ref) {
   const props = useThemeProps({ props: inProps, name: 'MuiScopedCssBaseline' });
-  const { className, component = 'div', ...other } = props;
+  const { className, component = 'div', enableColorScheme = false, ...other } = props;

   const ownerState = {
     ...props,
```

This should help, we don't want to spread it on the DOM.

Reply:
And note that you can access it in the styles via `ownerProps.enableColorScheme`